src/wmnds/components/lists/ordered-list/_ordered-list.scss

```
@@ -0,0 +1,20 @@
+.wmnds-ordered-list {
+  counter-reset: item;
+  padding-inline-start: 0;
+
+  ol {
+    counter-reset: item;
+    padding-inline-start: $size-sm * 2;
```

```suggestion
    padding-inline-start: $size-lg;
```
I'd recommend changing this to the recommended. 
By multiplying by 2 we end up creating a new "size" which doesn't exist (1.5rem).

It's best to stick with our defined sizes.